core/main/src/service/user_grants.rs

```diff
@@ -1357,7 +1401,7 @@ mod tests {
                     }],
                 },
             ]);
-            let caller_session: CallerSession = ctx.clone().into();
+            let caller_session: CallerSession = CallerSession::default();
```

Comment:
Passing None for app_id and session Id as caller_session for evaluate_options() ?  Why do we need to pass these default values?  
We have eliminated this dependency in Ripple 1.0.  Create an R2 Sync ticket to track those changes.

Reply:
This changes is in unit test. Passing None for app_id and session_id is to skip checking on app state because get_app_state is not mock to return foreground state therefore a few unit tests are failing.